### ISSUE TITLE
Getting empty data on Zoro and other providers

### ISSUE THREAD

**Issue by itz-ankit** [consumet/api.consumet.org#649]
### Describe the bug

I'm s wunable to fetch streaming links data, it waorking pretty well yesterday. I checked today it isn't working. I don't know what's wrong maybe I should wait or I could really use an explanation right now.

### Steps to reproduce

Tried several /watch endpoints with several episode Ids but i'm getting empty data in all of them 
![image](https://github.com/user-attachments/assets/c8b6e63c-9a79-415d-b662-983766a2ad56)

### Expected behavior

```json
{
   "headers": {
        "Referer": "string",
        "watchsb": "string",
        "User-Agent": "string"
    },
   "sources": [
    {
        "url": "string",
        "quality": "string",
        "isM3U8": true
    }]
}
```

### Actual behavior

```json
{
    "sources": [],
    "subtitles": []
}
```

### Additional context

Thank you so much for everything though, really appreciate it and I also heard that hianime did make some changes for this to occur

**Comment by Subhajeetch** [comment]
Bro just copy pasted mine 😭🙏

**Comment by itz-ankit** [comment]
> Bro just copy pasted mine 😭🙏

Yeah wanted to let them know here as well as i didn't see anyone bring it up :') 
My apologies tho

**Comment by Subhajeetch** [comment]
> > Bro just copy pasted mine 😭🙏
> 
> Yeah wanted to let them know here as well as i didn't see anyone bring it up :') My apologies tho

Np dude 🫂

**Comment by Frofter** [comment]
> ### Describe the bug
> I'm unable to fetch streaming links data, it was working pretty well yesterday. I checked today it isn't working. I don't know what's wrong maybe I should wait or I could really use an explanation right now.
> 
> ### Steps to reproduce
> Tried several /watch endpoints with several episode Ids but i'm getting empty data in all of them ![image](https://private-user-images.githubusercontent.com/116746318/394920072-c8b6e63c-9a79-415d-b662-983766a2ad56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3Njc1ODIsIm5iZiI6MTczNDc2NzI4MiwicGF0aCI6Ii8xMTY3NDYzMTgvMzk0OTIwMDcyLWM4YjZlNjNjLTlhNzktNDE1ZC1iNjYyLTk4Mzc2NmEyYWQ1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyMVQwNzQ4MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMjhiYjQ1Y2FlOGRiMDVhZDhjMmU1OTljODY5MzI2ZWM2YzdmOWU2YjU3YzNiNDlkMzZjY2I0ZDdjY2NkYmI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.syTml3o30784nA28T0A7CSMxsjestqEhnc67sqqY2VI)
> 
> ### Expected behavior
> ```json
> {
>    "headers": {
>         "Referer": "string",
>         "watchsb": "string",
>         "User-Agent": "string"
>     },
>    "sources": [
>     {
>         "url": "string",
>         "quality": "string",
>         "isM3U8": true
>     }]
> }
> ```
> 
> ### Actual behavior
> ```json
> {
>     "sources": [],
>     "subtitles": []
> }
> ```
> 
> ### Additional context
> Thank you so much for everything though, really appreciate it and I also heard that hianime did make some changes for this to occur

its working fine for me though i only checked the gogoanime one (didn`t tried the others) and also most other anime websites (even some popular one) are down because of copyright or either stopped uploading new content same goes for asian dramas websites

**Comment by itz-ankit** [comment]
> > ### Describe the bug
> > I'm unable to fetch streaming links data, it was working pretty well yesterday. I checked today it isn't working. I don't know what's wrong maybe I should wait or I could really use an explanation right now.
> > ### Steps to reproduce
> > Tried several /watch endpoints with several episode Ids but i'm getting empty data in all of them ![image](https://private-user-images.githubusercontent.com/116746318/394920072-c8b6e63c-9a79-415d-b662-983766a2ad56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3Njc1ODIsIm5iZiI6MTczNDc2NzI4MiwicGF0aCI6Ii8xMTY3NDYzMTgvMzk0OTIwMDcyLWM4YjZlNjNjLTlhNzktNDE1ZC1iNjYyLTk4Mzc2NmEyYWQ1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyMVQwNzQ4MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMjhiYjQ1Y2FlOGRiMDVhZDhjMmU1OTljODY5MzI2ZWM2YzdmOWU2YjU3YzNiNDlkMzZjY2I0ZDdjY2NkYmI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.syTml3o30784nA28T0A7CSMxsjestqEhnc67sqqY2VI)
> > ### Expected behavior
> > ```json
> > {
> >    "headers": {
> >         "Referer": "string",
> >         "watchsb": "string",
> >         "User-Agent": "string"
> >     },
> >    "sources": [
> >     {
> >         "url": "string",
> >         "quality": "string",
> >         "isM3U8": true
> >     }]
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ### Actual behavior
> > ```json
> > {
> >     "sources": [],
> >     "subtitles": []
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ### Additional context
> > Thank you so much for everything though, really appreciate it and I also heard that hianime did make some changes for this to occur
> 
> its working fine for me though i only checked the gogoanime one (didn`t tried the others) and also most other anime websites (even some popular one) are down because of copyright or either stopped uploading new content same goes for asian dramas websites

The zoro endpoint is still not working unfortunately
